core/src/main/java/com/linecorp/armeria/client/endpoint/WeightRampingUpStrategy.java

```diff
@@ -187,8 +187,8 @@ private void updateEndpoints(List<Endpoint> newEndpoints) {
             if (rampingUpTaskWindowNanos > 0) {
                 // Check whether we can ramp up with the previous ramped up endpoints which are at the last
                 // of the rampingUpEndpointsEntries.
+                final Set<EndpointAndStep> newlyAddedEndpoints = filterOldEndpoints(newEndpoints);
```

Comment:
This affects existing ramping up(removing endpoints if disappeared). I'm curious if it's still ok to use `newEndpoints` for line [203](https://github.com/line/armeria/blob/13e6d13546ea60cf65e148e9e0e5771f518608e1/core/src/main/java/com/linecorp/armeria/client/endpoint/WeightRampingUpStrategy.java#L203), I think it should be ok because `filterOldEndpoints(unhandledNewEndpoints)` is called at `updateWeightAndStep`.

Reply:
> I'm curious if it's still ok to use newEndpoints for line [203](https://github.com/line/armeria/blob/13e6d13546ea60cf65e148e9e0e5771f518608e1/core/src/main/java/com/linecorp/armeria/client/endpoint/WeightRampingUpStrategy.java#L203) I think it should be ok 

So do I. 😆